zboss/production/include/addons/zboss_api_addons.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Nordic Semiconductor ASA
+ * Copyright (c) 2021 Nordic Semiconductor ASA
```

Comment:
This has to be `2020-2021` or nothing. We are waiting for the feedback.

Reply:
I removed year from Nordic license headers